specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -12342,34 +12342,6 @@
       },
       "x-ms-azure-resource": true
     },
-    "ExtendedLocationType": {
```

Comment:
HI @Sandido so where's `ExtendedLocationType` and `ExtendedLocation` for Compute RP resources? are they in disk.json?  It's recommended to put under /common folder if something common like azure-rest-api-specs\specification\storage\resource-manager\Microsoft.Storage\stable\2019-06-01\common.json

Reply:
and will this change in master branch together with `2020-12-01` releaes?